### PR TITLE
Feature oi monitor

### PR DESCRIPTION
**Pull request by jhinspeter**

### CODE REVIEW

File: src/main/java/frc/robot/events/LeftOperatorStickBackward.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import frc.robot.subsystems.AcquisitionRoller.RollerState;
+
+public class LeftOperatorStickBackward implements Event {
+
+    public boolean check() {
+        return (OI.operatorStick.getRawAxis(OI.NykoController.leftYAxis) > Constants.CloseLoopJoystickDeadband);
+    }
+
+    public Command getCommand() {
+        return new SetRoller(RollerState.EJECT);
```
Comment (Team138Entropy) [inline src/main/java/frc/robot/events/LeftOperatorStickBackward.java]: Switch to STOP

File: src/main/java/frc/robot/events/LeftOperatorStickForward.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import frc.robot.subsystems.AcquisitionRoller.RollerState;
+
+public class LeftOperatorStickForward implements Event {
+
+    public boolean check() {
+        return -OI.operatorStick.getRawAxis(OI.NykoController.leftYAxis) > Constants.CloseLoopJoystickDeadband;
+    }
+
+    public Command getCommand() {
+        return new SetRoller(RollerState.INTAKE);
```
Comment (Team138Entropy) [inline src/main/java/frc/robot/events/LeftOperatorStickForward.java]: Switch to TOGGLE

File: src/main/java/frc/robot/subsystems/AcquisitionRoller.java [hunk 3]
```diff
@@ -3,46 +3,76 @@
...
+            case INTAKE:
+                on = true;
+                rollerTalon.setInverted(false);
+                break;
+            case EJECT:
+                on = true;
+                rollerTalon.setInverted(true);
+                break;
+            case STOP:
+                break;
```
Comment (Team138Entropy) [inline src/main/java/frc/robot/subsystems/AcquisitionRoller.java]: set on = false for clarity

File: src/main/java/frc/robot/Sensors.java [hunk 4]
```diff
@@ -62,6 +93,7 @@ public static void resetEncoders() {
 	 public static void updateSmartDashboard(){
 		SmartDashboard.putBoolean("Practice Bot", isPracticeBot());
 		SmartDashboard.putBoolean("Cargo Present", isCargoPresent());
```
[the remark this answers is not in the document]
Comment (jhinspeter) [reply src/main/java/frc/robot/Sensors.java]: Done

File: src/main/java/frc/robot/OI.java [hunk 5]
```diff
@@ -30,72 +33,75 @@
...
+    static double LastY = 0;
 
 	static Button homeElevatorButton = new JoystickButton(operatorStick, NykoController.middle11);
 	static Button elevateToLevel1    = new JoystickButton(operatorStick, NykoController.button1);
 	static Button elevateToLevel2    = new JoystickButton(operatorStick, NykoController.button2);
 	static Button elevateToLevel3    = new JoystickButton(operatorStick, NykoController.button4);
 	static Button rotateTurretLeft   = new JoystickButton(operatorStick, NykoController.leftBumper);
 	static Button rotateTurretRight  = new JoystickButton(operatorStick, NykoController.rightBumper);
-	static Button rollerTestButton   = new JoystickButton(operatorStick, NykoController.leftStick);
+    static Button rollerTestButton   = new JoystickButton(operatorStick, NykoController.leftStick);
```
[the remark this answers is not in the document]
Comment (jhinspeter) [reply src/main/java/frc/robot/OI.java]: Done

File: src/main/java/frc/robot/commands/DeployCargo.java [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import frc.robot.Constants;
+
+public class DeployCargo extends CommandGroup {
+
+    public DeployCargo() {
+        addSequential(new UnlatchCargo());
+        addSequential(new TranslateManipulator(Constants.EXTEND));
+        addSequential(new Wait(0.150));
```
[the remark this answers is not in the document]
Comment (jhinspeter) [reply src/main/java/frc/robot/commands/DeployCargo.java]: Added constants

File: src/main/java/frc/robot/commands/ToggleRollerPistons.java [hunk 7]
```diff
@@ -11,6 +11,7 @@ public ToggleRollerPistons() {
     }
 
     protected void execute() {
+        System.out.println("trying to use piston");
```
[the remark this answers is not in the document]
Comment (jhinspeter) [reply src/main/java/frc/robot/commands/ToggleRollerPistons.java]: Removed

File: src/main/java/frc/robot/commands/Wait.java [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+import frc.robot.Constants;
+import frc.robot.Robot;
+import frc.robot.subsystems.Elevator.ElevatorTarget;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class Wait extends Command {
+	
+	private double commandTimeoutSeconds = 7;
```
[the remark this answers is not in the document]
Comment (jhinspeter) [reply src/main/java/frc/robot/commands/Wait.java]: Removed default value. (Set to 0)

File: src/main/java/frc/robot/events/LeftOperatorStickBackward.java (hunk 1 above)
Comment (jhinspeter) [reply src/main/java/frc/robot/events/LeftOperatorStickBackward.java]: Done

File: src/main/java/frc/robot/events/LeftOperatorStickForward.java (hunk 2 above)
Comment (jhinspeter) [reply src/main/java/frc/robot/events/LeftOperatorStickForward.java]: Added ReverseRoller command

File: src/main/java/frc/robot/subsystems/AcquisitionRoller.java (hunk 3 above)
Comment (jhinspeter) [reply src/main/java/frc/robot/subsystems/AcquisitionRoller.java]: Done

**Pull request merged**